css/customstyles.css

```diff
@@ -932,16 +932,6 @@ span.soft {
     padding:10px;
 }
 
-/* this part adds an icon after external links, using FontAwesome*/
-a[href^="http://"]:after, a[href^="https://"]:after {
```

Comment:
If we still decided to keep it, i think it would be in the lines of
```
a[href^="http://"]:after, a[href^="https://"]:not(#fork-me):after {
...
```

Reply:
I'll save this regardless, so that I may have learned something, if nothing else.

Reply:
:warning: I have not tested this. It's fresh from the docs:
https://www.w3.org/TR/selectors-3/#negation